### PR TITLE
[readme] Clarify runtime installation

### PR DESCRIPTION
**Pull request by ryanrhee**
To avoid situations like https://github.com/pre-commit/pre-commit.com/pull/405

### CODE REVIEW

**Comment by ryanrhee** [comment]
@asottile I'm not sure if this is necessary, wdyt?

**Comment by asottile** [comment]
I don't really think this is necessary, it mentions already that it provides a `shellcheck` binary 🤷

File: README.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 
 # shellcheck-py
 
-A python wrapper to provide a pip-installable [shellcheck] binary.
+Run shellcheck in pre-commit even if the host machine doesn't have it installed!
```
Comment (asottile) [inline README.md:6]: it also works even without pre-commit, (I find it way more convenient than fishing through the binaries page(s))

**Comment by ryanrhee** [comment]
ya agreed, gonna close.

**Review by asottile — APPROVED** [review]
-0.0 essentially

File: README.md (hunk 1 above)
Comment (ryanrhee) [reply README.md:6]: ah yeah good point!

**Pull request closed without merging**